README.md

```diff
@@ -155,42 +174,52 @@ A wide variety of examples showcasing what Nutrient Web SDK can do can be found
+
+- [Auto-Resizing Comment Threads on Zoom](https://www.nutrient.io/playground?p=eyJ2IjoxLCJqcyI6IlBTUERGS2l0LmxvYWQoe1xuICAuLi5iYXNlT3B0aW9ucyxcbiAgdGhlbWU6IFBTUERGS2l0LlRoZW1lLkRBUkssXG59KS50aGVuKChpbnN0YW5jZSkgPT4ge1xuICBsZXQgem9vbUZhY3RvciA9IDE7XG5cbiAgaW5zdGFuY2UuYWRkRXZlbnRMaXN0ZW5lcihcbiAgICBcInZpZXdTdGF0ZS5jaGFuZ2VcIixcbiAgICAodmlld1N0YXRlLCBwcmV2aW91c1ZpZXdTdGF0ZSkgPT4ge1xuICAgICAgY29uc3Qgem9vbUxldmVsID0gdmlld1N0YXRlLnpvb20uem9vbU1vZGU7XG4gICAgICBjb25zdCBwcmV2Wm9vbUxldmVsID0gcHJldmlvdXNWaWV3U3RhdGUuem9vbS56b29tTW9kZTtcblxuICAgICAgLy8gQ2FsY3VsYXRlIHpvb20gY2hhbmdlIHJhdGlvXG4gICAgICBsZXQgem9vbUNoYW5nZSA9IHpvb21MZXZlbCAvIHByZXZab29tTGV2ZWw7XG5cbiAgICAgIC8vIEZhbGxiYWNrIHRvIGRlZmF1bHQgem9vbSBzdGVwIGlmIHRoZSBjYWxjdWxhdGlvbiBmYWlscy5cbiAgICAgIC8vIFRoaXMgY2FuIGhhcHBlbiBpZiB0aGUgcHJldmlvdXMgem9vbSB3YXMgc2V0IHRvIFwiQVVUT1wiXG4gICAgICBpZiAoaXNOYU4oem9vbUNoYW5nZSkpIHtcbiAgICAgICAgem9vbUNoYW5nZSA9IHZpZXdTdGF0ZS56b29tU3RlcDtcbiAgICAgIH1cblxuICAgICAgLy8gVXBkYXRlIG92ZXJhbGwgem9vbSBmYWN0b3JcbiAgICAgIHpvb21GYWN0b3IgKj0gem9vbUNoYW5nZTtcblxuICAgICAgLy8gQXBwbHkgem9vbSBvbiB2aXNpYmxlIGNvbW1lbnQgdGhyZWFkc1xuICAgICAgY29uc3QgY29tbWVudHMgPVxuICAgICAgICBpbnN0YW5jZS5jb250ZW50RG9jdW1lbnQucXVlcnlTZWxlY3RvckFsbChcIi5QU1BERktpdC1Db21tZW50LVRocmVhZFwiKTtcbiAgICAgIGNvbW1lbnRzLmZvckVhY2goKGNvbW1lbnQpID0%252BIHtcbiAgICAgICAgY29tbWVudC5zdHlsZS50cmFuc2Zvcm0gPSBgc2NhbGUoJHt6b29tRmFjdG9yfSlgO1xuICAgICAgfSk7XG4gICAgfSxcbiAgKTtcblxuICAvLyBPYnNlcnZlIGZvciBuZXcgY29tbWVudHMgYmVpbmcgYWRkZWQgdG8gRE9NIGFuZCBhcHBseSB6b29tIHRvIHRoZWlyIHBvcG92ZXJzXG4gIGNvbnN0IGFwcGx5Wm9vbVRvQ29tbWVudCA9IChtdXRhdGlvbkxpc3QpID0%252BIHtcbiAgICBtdXRhdGlvbkxpc3QuZm9yRWFjaCgobXV0YXRpb24pID0%252BIHtcbiAgICAgIG11dGF0aW9uLmFkZGVkTm9kZXMuZm9yRWFjaCgobm9kZSkgPT4ge1xuICAgICAgICBpZiAobm9kZS5tYXRjaGVzICYmIG5vZGUubWF0Y2hlcyhcIi5CYXNlbGluZVVJLVBvcnRhbFwiKSkge1xuICAgICAgICAgIGNvbnN0IGNvbW1lbnRUaHJlYWQgPSBub2RlLnF1ZXJ5U2VsZWN0b3IoXCIuUFNQREZLaXQtQ29tbWVudC1UaHJlYWRcIik7XG4gICAgICAgICAgaWYgKGNvbW1lbnRUaHJlYWQpIHtcbiAgICAgICAgICAgIGxldCBwYXJlbnQgPSBjb21tZW50VGhyZWFkLnBhcmVudEVsZW1lbnQ7XG4gICAgICAgICAgICB3aGlsZSAoXG4gICAgICAgICAgICAgIHBhcmVudCAmJlxuICAgICAgICAgICAgICAhcGFyZW50LmNsYXNzTGlzdC5jb250YWlucyhcIkJhc2VsaW5lVUktUG9wb3Zlci1Db250ZW50XCIpXG4gICAgICAgICAgICApIHtcbiAgICAgICAgICAgICAgcGFyZW50ID0gcGFyZW50LnBhcmVudEVsZW1lbnQ7XG4gICAgICAgICAgICB9XG4gICAgICAgICAgICBpZiAocGFyZW50KSB7XG4gICAgICAgICAgICAgIHBhcmVudC5zdHlsZS50cmFuc2Zvcm0gPSBgc2NhbGUoJHt6b29tRmFjdG9yfSlgO1xuICAgICAgICAgICAgICBwYXJlbnQuc3R5bGUudHJhbnNmb3JtT3JpZ2luID0gXCJ0b3BcIjtcbiAgICAgICAgICAgIH1cbiAgICAgICAgICB9XG4gICAgICAgIH1cbiAgICAgIH0pO1xuICAgIH0pO1xuICB9O1xuXG4gIGNvbnN0IG9ic2VydmVyID0gbmV3IE11dGF0aW9uT2JzZXJ2ZXIoYXBwbHlab29tVG9Db21tZW50KTtcbiAgb2JzZXJ2ZXIub2JzZXJ2ZShpbnN0YW5jZS5jb250ZW50RG9jdW1lbnQsIHtcbiAgICBjaGlsZExpc3Q6IHRydWUsXG4gICAgc3VidHJlZTogdHJ1ZSxcbiAgfSk7XG5cbiAgcmV0dXJuIGluc3RhbmNlO1xufSk7XG4iLCJjc3MiOiIvKiBBZGQgeW91ciBDU1MgaGVyZSAqL1xuXHQiLCJzZXR0aW5ncyI6eyJmaWxlTmFtZSI6ImJhc2ljLnBkZiJ9fQ%253D%253D)

```

Comment:
I think instead of using mutation observer you can rely on comments creation event. Should be more performant.